homeassistant/components/google/strings.json

```diff
@@ -39,7 +39,9 @@
     "step": {
       "init": {
         "data": {
-          "calendar_access": "Home Assistant access to Google Calendar"
+          "calendar_access": "Home Assistant access to Google Calendar",
+          "read_write": "Read/Write access (can create events)",
+          "read_only": "Read-only access"
```

Comment:
This isn't correct. These aren't fields, these are values of the `calendar_access` field.

For more information on backend translation, see the developer documentation: https://developers.home-assistant.io/docs/internationalization/core

Also, please be sure to test your PRs before making them 👍 

../Frenck

Reply:
but according to what I understood from the document, I have to add the new strings in the `strings.json` file, and Lokalise will handle the rest.

these are the two fields that I am trying to translate:
![image](https://github.com/user-attachments/assets/0ea0c035-6882-4a7f-b1f0-c79edfd4f290)

Reply:
Yes, it's clear what you're trying to do, but either way, the code doesn't work? e.g. if you change the string values in the json file, they won't show up in the UI, right? he proposed strings change are for fields, but these are changes for values for a field. This is what i was trying to save above that this does not seem like it would actually work.